### PR TITLE
Implemented availability logging and graph display

### PR DESCRIPTION
**Pull request by hrideshmg**
image

### CODE REVIEW

File: static/graph.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        label: "Availability",
+        data: health_statuses,
+        borderWidth: 1,
+      },
+    ],
+  },
+  options: {
+    scales: {
+      y: {
+        beginAtZero: true,
```
Comment (gopavasanth) [inline static/graph.js:18]: Also, add these properties here:

      y: {
        beginAtZero: true,
        ticks: {
          stepSize: 1,
          min: 0,
          max: 1
        }
      },

Comment (gopavasanth) [reply static/graph.js:18]: More feedback provided here: https://phabricator.wikimedia.org/T365857

**Comment by varun-s22** [comment]
can you add a screenshot as well?

**Comment by gopavasanth** [comment]
image

@varun-s22  for your reference.  The patch looks good to me. Thanks a lot!! @hrideshmg and @hrideshmg!!
@sohomdatta1 any thoughts here? Shall we merge this PR?

**Review by gopavasanth — APPROVED** [review]

**Comment by sohomdatta1** [comment]
Lets merge :)

**Comment by sohomdatta1** [comment]
Infra work done, the tool is now powered by buildpacks :)

**Comment by gopavasanth** [comment]
Live at: https://tool-watch.toolforge.org/

**Pull request merged**